bitnami/rabbitmq-cluster-operator/Chart.yaml

```
@@ -37,4 +37,4 @@ maintainers:
 name: rabbitmq-cluster-operator
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/rabbitmq-cluster-operator
-version: 4.1.0
+version: 4.1.1
```

Let's bump the minor version as this is introducing new parameters:

```suggestion
version: 4.2.0
```